### PR TITLE
Add fullname for users

### PR DESCRIPTION
**Pull request by pontus**
Add full name (`fullname`) for users, will be needed for e.g. certificates.

### CODE REVIEW

**Review by jonandernovella — APPROVED** [review]
Great!

File: app/utils/database.py [hunk 1]
```diff
@@ -1270,12 +1280,41 @@ def migrate_5_to_6():
...
-            version=5,
+            version=6,
             comment="Remove password_hash from hbuser",
             applied=datetime.now(),
         ).execute()
 
 
+def migrate_6_to_7():
+    """
+    Migrate between schema version 5 and 6.
```
Comment (HenrikeW) [inline app/utils/database.py:1291]: probably a wrong comment here, with regard to the function's name?

File: app/utils/data_access.py [hunk 2]
```diff
@@ -439,7 +445,7 @@ def update_user(form, user_uuid=None):
         return {"status": "error", "message": f"malformed request: {form}"}
 
     # Check permissions - allow users to update e-mail only
-    if not (user.is_admin or user.is_manager) or (
+    if not (user.is_admin or user.is_manager) and (
```
Comment (HenrikeW) [inline app/utils/data_access.py:448]: I don't really understand why this needed to be changed.

**Review by HenrikeW — APPROVED** [review]

File: app/utils/database.py (hunk 1 above)
Comment (pontus) [reply app/utils/database.py:1291]: Yes, :eyes: .

**Pull request merged**